add docs meta descriptions (#32771)
hashicorp/terraform: website/docs/language/modules/develop/providers.mdx

---
page_title: Providers Within Modules - Configuration Language
description: >-

>   Use providers within Terraform modules. Learn about version constraints, aliases, implicit inheritance, and passing providers explicitly in Terraform modules.

Reviewer: ```suggestion
  Use providers within Terraform modules. Learn about version constraints, aliases, implicit inheritance, and passing providers to Terraform modules.
```

is that still right?
